sale_last_price_info/models/product.py

```diff
@@ -0,0 +1,28 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _compute_last_sale(self):
+        """Get last sale price, last sale date and last customer"""
+        so_line_obj = self.env["sale.order.line"]
+        for product in self:
+            line = so_line_obj.search(
```

Comment:
Something has to be done here as this will generate as many queries as products in recordset.